docs/src/main/sphinx/eventlistener/http-event-listener.rst

```diff
@@ -0,0 +1,94 @@
+===================
+Http Event Listener
+===================
+
+The Http Event Listener plugin allows streaming of query events, encoded in
+JSON format, to an external service for further processing, by POSTing them
+to a specified URI.
+
+Rationale
+---------
+
+This event listener is a simple first step into better understanding the usage
+of a datalake using query events provided by Trino. These can provide CPU and memory
+usage metrics, what data is being accessed (with resolution down to specific columns),
+and meta-information about the query.
+
+Running this system decoupled from Trino means there will be no performance hit and no
+downtime for non-client-facing changes.
+
+Requirements
+------------
+
+To use this plugin, you need:
+
+* An HTTP/S service that will accept POST requests with a JSON body to a URI.
+* To set ``http-event-listener.ingest-uri`` in the event listener properties to
+  the URI from your service.
+* By default, this event event listener will not send any events. You must configure
+  which events you want to be sent in the :ref:`http_event_listener_configuration` section.
+
+.. _http_event_listener_configuration:
+
+Configuration
+-------------
+
+To configure the Http Event Listener plugin, create an event listener properties
+file in ``etc`` named ``event-listener.properties`` with the following contents
+as an example:
+
+.. code-block:: text
+
+    event-listener.name=http
+    http-event-listener.log-created=true
+    http-event-listener.connect-ingest-uri=<your ingest URI>
+
+If you have other event listeners implemented but wish to use this one make sure the
+name is set to ``http``. If you want to use multiple event listeners refer to
```

Comment:
Isn't setting `event-listener.name=http` always needed. No matter if there is one ore more implementations of event listeners?
Anyway I would skip this sentence. Example shows how to use it clearly enough.